cafebabel/templates/base.html

```diff
@@ -3,11 +3,28 @@
 <meta name=viewport content="width=device-width, minimum-scale=1, initial-scale=1, shrink-to-fit=no">
 <title>{% block meta_title %}pan-European news, features and opinions{% endblock %} – Cafebabel</title>
 <meta name=description content="{% block meta_description %}{% endblock %}">
+<!--Social-->
+<meta property="og:url" content="{{ request.url }}">
+<meta property="og:title" content="{{ self.meta_title() }}">
+<meta property="og:image" content="{% block meta_image_url %}{% endblock %}"> <!-- to discuss -->
+<meta property="og:type" content="{% block meta_type %}website{% endblock %}">
+<meta property="og:description" content="{{ self.meta_description() }}">
+<meta property="og:site_name" content="Cafébabel">
+<meta property="og:locale" content="{% block meta_language %}{% endblock %}">
+<meta property="article:author" content="{% block meta_author %}{% endblock %}">
+
+<meta name="twitter:card" content="summary">
+<meta name="twitter:title" content="{{ self.meta_title() }}">
+<meta name="twitter:description" content="{{ self.meta_description() }}">
+<meta name="twitter:image" content="{{ self.meta_image_url() }}">
+<meta name="twitter:url" content="{{ request.url }}">
```

Comment:
You *might* want to [factorize all that metacrap](https://adactio.com/journal/9881) but it costs some readability, I let you arbitrate :)